src/connectors/clear-refinements/connectClearRefinements.js

```diff
@@ -179,7 +179,7 @@ function getAttributesToClear({
 
   return transformItems(
     getRefinements(helper.lastResults || {}, helper.state, clearsQuery)
-      .map(refinement => refinement.attributeName)
+      .map(refinement => refinement.attribute)
```

Comment:
great catch! How long was this broken then?

Reply:
That wasn't broken. The internal implementation created a key called `attributeName`. I changed it to `attribute` to type this data structure more easily. I think it's less confusing this way.